apps/api/src/app/bridge/bridge.module.ts

```diff
@@ -13,11 +13,42 @@ import {
   UpsertControlValuesUseCase,
   UpsertPreferences,
   DeletePreferencesUseCase,
+  TierRestrictionsValidateUsecase,
 } from '@novu/application-generic';
-import { PreferencesRepository } from '@novu/dal';
+import { CommunityOrganizationRepository, PreferencesRepository } from '@novu/dal';
 import { SharedModule } from '../shared/shared.module';
 import { BridgeController } from './bridge.controller';
 import { USECASES } from './usecases';
+import { UpsertWorkflowUseCase } from '../workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase';
+import { PostProcessWorkflowUpdate } from '../workflows-v2/usecases/post-process-workflow-update';
+import { PatchStepUsecase } from '../workflows-v2/usecases/patch-step-data';
+import { OverloadContentDataOnWorkflowUseCase } from '../workflows-v2/usecases/overload-content-data';
+import { BuildStepDataUsecase } from '../workflows-v2/usecases/build-step-data';
+import {
+  BuildDefaultPayloadUsecase,
+  CollectPlaceholderWithDefaultsUsecase,
+  PrepareAndValidateContentUsecase,
+  ValidatePlaceholderUsecase,
+} from '../workflows-v2/usecases/validate-content';
+import { BuildAvailableVariableSchemaUsecase } from '../workflows-v2/usecases/build-variable-schema';
+import { ExtractDefaultValuesFromSchemaUsecase } from '../workflows-v2/usecases/extract-default-values-from-schema';
+import { HydrateEmailSchemaUseCase } from '../environments-v1/usecases/output-renderers/hydrate-email-schema.usecase';
+
+const SHARED_USE_CASES = [
```

Comment:
Since these aren't shared with anything else, we should move them to the PROVIDERS array.

Reply:
I planned to refactor this part and move it into a separate NestJS module so that both the Sync Module and the WorkflowV2 Module can use it. However, I'll postpone this for now since including it in this PR would slow down its progress.

For now, i will do as you suggested ;)